hack/cr-vector.yaml

```diff
@@ -3,6 +3,8 @@ kind: "ClusterLogging"
 metadata:
   name: "instance"
   namespace: "openshift-logging"
+  annotations:
+    logging.openshift.io/preview-vector-collector: enabled
```

Comment:
can remove this line, the feature gate is being removed in PR https://github.com/openshift/cluster-logging-operator/pull/1455